Source/Core/Common/HttpRequest.cpp

```diff
@@ -69,7 +73,16 @@ HttpRequest::Response HttpRequest::Post(const std::string& url, const std::strin
                        reinterpret_cast<const u8*>(payload.data()), payload.size());
 }
 
-HttpRequest::Impl::Impl(std::chrono::milliseconds timeout_ms)
+int HttpRequest::Impl::CurlProgressCallback(HttpRequest::Impl* impl, double dlnow, double dltotal,
```

Comment:
I think you don't need to explicitly write 'HttpRequest::' again for `impl` since this is a member function.